src/styles/mixins.scss

```diff
@@ -1,32 +1,8 @@
-@mixin box-shadow($shadow...) {
-  -webkit-box-shadow: $shadow;
-     -moz-box-shadow: $shadow;
-          box-shadow: $shadow;
-}
-
-@mixin box-sizing($sizing...) {
-  -webkit-box-sizing: $sizing;
-     -moz-box-sizing: $sizing;
-          box-sizing: $sizing;
-}
-
-@mixin border-radius($radius...) {
-  -webkit-border-radius: $radius;
-     -moz-border-radius: $radius;
-          border-radius: $radius;
-}
-
-@mixin border-top-left-radius($radius...) {
-  -webkit-border-top-left-radius: $radius;
-     -moz-border-radius-topleft:  $radius;
-          border-top-left-radius: $radius;
-}
-
-@mixin border-top-right-radius($radius...) {
-  -webkit-border-top-right-radius: $radius;
-     -moz-border-radius-topright:  $radius;
-          border-top-right-radius: $radius;
-}
+@mixin box-shadow($shadow...) { box-shadow: $shadow }
+@mixin box-sizing($sizing...) { box-sizing: $sizing }
+@mixin border-radius($radius...) { border-radius: $radius }
+@mixin border-top-left-radius($radius...) { border-top-left-radius: $radius }
+@mixin border-top-right-radius($radius...) { border-top-right-radius: $radius }
```

Comment:
If we are going to delete these, there is no point making them mixins.

Reply:
Resolved by removing the mixins entirely..